### PR TITLE
Install amp-form CSS and cleanup previous renders during submission

### LINKED ISSUE #3587: Cleanup rendered templates during submissions in forms
A follow up to #3583 specifically to [this part](https://github.com/ampproject/amphtml/pull/3583#discussion-diff-67188147)

### PR DESCRIPTION
**Pull request by mkhatib**
Closes #3587

ITI: #3343

### CODE REVIEW

File: extensions/amp-form/0.1/amp-form.js [hunk 1]
```diff
@@ -124,20 +126,25 @@ export class AmpForm {
...
     this.submitButtons_.forEach(button => {
-      if (state == FormState_.SUBMITTING) {
+      if (newState == FormState_.SUBMITTING) {
         button.setAttribute('disabled', '');
       } else {
         button.removeAttribute('disabled');
       }
     });
+
+    if (newState == FormState_.SUBMITTING) {
```
Comment (jridgewell) [inline extensions/amp-form/0.1/amp-form.js]: I think this'd be more appropriate before setting the state to submitting (before calling `#setState`)

File: extensions/amp-form/0.1/amp-form.js [hunk 2]
```diff
@@ -148,8 +155,6 @@ export class AmpForm {
   renderTemplate_(state, data = {}) {
```
Comment (jridgewell) [inline extensions/amp-form/0.1/amp-form.js]: No need for a `state` param, we have `#state` now.

Comment (mkhatib) [inline extensions/amp-form/0.1/amp-form.js]: Done

File: extensions/amp-form/0.1/amp-form.js (hunk 1 above)
Comment (mkhatib) [inline extensions/amp-form/0.1/amp-form.js]: Done.

**Comment by mkhatib** [comment]
PTAL

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**